### PR TITLE
Elo calculator

### PR DESCRIPTION
**Pull request by Isylwin**

### CODE REVIEW

File: common/src/main/nl/jft/common/util/elo/EloExpectationLoader.java [hunk 1]
```diff
@@ -0,0 +1,12 @@
+package nl.jft.common.util.elo;
+
+import java.util.List;
+
+/**
+ * @author Lesley
+ */
+public interface EloExpectationLoader {
```
Comment (TehMessiah) [inline common/src/main/nl/jft/common/util/elo/EloExpectationLoader.java]: CRITICAL Annotate the "EloExpectationLoader" interface with the @FunctionalInterface annotation (sonar.java.source not set. Assuming 8 or greater.) [rule](http://92.110.24.119:9000/coding_rules#rule_key=squid%3AS1609)

File: common/src/main/nl/jft/common/util/elo/EloCalculationStrategy.java [hunk 2]
```diff
@@ -0,0 +1,19 @@
+package nl.jft.common.util.elo;
+
+/**
+ * An {@code EloCalculationStrategy} is a strategy to calculating a new Elo-rating based on the given parameters.
+ *
+ * @author Oscar de Leeuw
+ */
+public interface EloCalculationStrategy {
```
Comment (TehMessiah) [inline common/src/main/nl/jft/common/util/elo/EloCalculationStrategy.java]: CRITICAL Annotate the "EloCalculationStrategy" interface with the @FunctionalInterface annotation (sonar.java.source not set. Assuming 8 or greater.) [rule](http://92.110.24.119:9000/coding_rules#rule_key=squid%3AS1609)

**Comment by TehMessiah** [comment]
SonarQube analysis reported 2 issues
* CRITICAL 2 critical

Watch the comments in this conversation to review them.

**Comment by Isylwin** [comment]
retest this please

**Pull request merged**